pypeclub/OpenPype
# Houdini: Implement Redshift Proxy Export

Tested with Houdini 19.0.589 + Redshift 3.5.01

## Brief description

Allows to extract a Redshift Proxy `.rs` file from Houdini that can (currently) be loaded in OpenPype into Maya with its Redshift Proxy loader.

## Testing notes:

1. Launch Houdini with Redshift, export a proxy
2. Load proxy into Maya, ensure it renders fine.
